submissions/xT0PERx/js-dom/visuality.css

```diff
@@ -0,0 +1,99 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Courier New", Courier, monospace;
+}
+.menu-list {
+  list-style-type: none;
+  padding: 5px 0px 5px 0px;
+  min-width: 100px;
+}
+
+.menu-list li:active {
+  background-color: rgb(223, 20, 20);
+}
+
+.menu-title {
+  padding: 20px 20px 20px 20px;
+  border: solid 0.3em;
+  border-color: silver;
+  font-weight: bold;
+  margin: 0.5em;
+}
+
+.menu-title p {
+  text-align: end;
+  padding: 25px 25px 25px 25px;
+  margin: 1em;
+  background-color: lightgray;
+}
+
+.menu-item-1,
+.menu-item-2,
+.menu-item-3,
+.menu-item-4,
+.menu-item-5 {
+  padding: 20px 20px 20px 20px;
+  background-color: rgba(182, 160, 90, 0.486);
+  border-bottom: solid 0.2em;
+  border-radius: 30px 0 0 30px;
+  border-bottom-color: silver;
+  cursor: pointer;
+  margin: 1em;
+}
+
+.img-content {
+  width: 200px;
+  height: 200px;
+  float: right;
+  margin: 7px 0 7px 7px;
+}
+
+.content-style {
+  display: flex;
+  flex-flow: row wrap;
+  border: solid 0.3em;
+  border-color: rgba(182, 160, 90, 0.486);
+  margin: 0.5em 10px 0.5em 10px;
```

Comment:
[Read mor](https://developer.mozilla.org/en-US/docs/Web/CSS/padding)e about short entry, margins have the same behavior

Reply:
Fixed

Reply:
Not fixed here